### ISSUE TITLE
After default member fields are migrated in EE4, undefined index to the url member field

### ISSUE THREAD

**Issue by matthewjohns0n** [ExpressionEngine/ExpressionEngine#45]
**Description of the problem**
When upgrading a site that has content in the `url` member field (and it was in use), it is migrated to a new custom member field. When a field with "url" in it is used, for example the `{url_title}` variable, ExpressionEngine tries to set the member_url variable in Simple_variable.php at the top of the `_url()` function. It seems clear it is being set at the top of the function so it can be used in some of the later `else` cases. However, sometimes `$mfields['url']` is set, but `$data['m_field_id_'.$mfields['url'][0]]` is not set, and PHP gets sad. PHP thows a notice that m_field_id_12 is an undefined index. See error below.

**How To Reproduce**
1. Use the default member field `{url}` in ExpressionEngine 3
2. Upgrade site from ExpressionEngine 3 to 4 (or continue to 5).
3. Use the `{url_title}` in a context where the member fields are not loaded. I believe this is any channel entries loop.
4. See error below.

**Error Messages**
<pre><code>
Notice
Undefined index: m_field_id_12
ee/legacy/libraries/channel_entries_parser/components/Simple_variable.php, line 296

Severity: E_NOTICE
</code></pre>

**Environment Details:**
 - Version: 5.0.1
 - PHP Version: 7.1
 - MySQL Version: 5.7
 - OS: Docker running on a Mac
 - Web Server: nginx

**Possible Solution**
It also needs to be verified that `$data['m_field_id_'.$mfields['url'][0]]` is set before attempting to use it. PR coming.

**Comment by kevincupp** [comment]
I'm unable to reproduce this, maybe there are some steps missing to reproduce? Here's what I tried:

1. Installed a fresh copy of latest EE3 with default theme
2. Filled in the URL in my profile
3. Went ahead and created a test template with the following:
```
{exp:channel:entries}
{url_title}
{/exp:channel:entries}
```
4. Upgraded to EE5, double-checked the DB that the member field was converted correctly
5. Went to visit template, showed all my URL titles. Tried this:
```
{exp:channel:entries disable="member_data"}
{url_title}
{/exp:channel:entries}
```
6. Still no PHP error
Let me know what else I can do to reproduce.
